<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChineseDictionary</class>
 <widget class="QMainWindow" name="ChineseDictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChineseDictionary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="chineseWordLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="russianWordLineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="hskLineEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chineseWordLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chinese</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="russianWordLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Russian</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hskLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HSK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="chineseToRussianButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cn -&gt; Ru</string>
    </property>
   </widget>
   <widget class="QPushButton" name="russianToChineseButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ru -&gt; Cn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="russianWordsLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>730</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="examplesLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>730</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pinyinWordLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pinyinWordLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pinyin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pinyinButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pinyin</string>
    </property>
   </widget>
   <widget class="QLabel" name="monotonePinyinWordLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monotone Pinyin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="monotonePinyinWordLineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman No9 L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="monotonePinyinButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monotone</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
